tests/rules/BUILD

```diff
@@ -0,0 +1,8 @@
+# Simply ensures the file supposed to be downloaded by the gce_file
+# test rule was actually downloaded
+sh_test(
```

Comment:
Why not use a file_test (https://github.com/bazelbuild/bazel/blob/master/tools/build_rules/test_rules.bzl#L306)?